light-client/src/client.rs

```diff
@@ -317,7 +317,7 @@ impl<DB: Database, R: ConsensusRpc + 'static> Client<DB, R> {
                 NodeError::ConsensusSyncError(err) => match err
                     .downcast_ref()
                     .ok_or(err.to_string())
-                    .map_err(|err| eyre!(err))?
+                    .map_err(|err| anyhow::anyhow!(err))?
```

Comment:
nitpick: for consistency, I would prefer to use directly `anyhow!`, instead of `anyhow::anyhow!`  in this file and in [light-client/src/config/checkpoints.rs](https://github.com/ethereum/trin/pull/1092/files#diff-ca43eb11df384433c0830c3335fa7b3ddca950da5a8067f9ac20d771d9a0bc46):

```suggestion
                    .map_err(|err| anyhow!(err))?
```